README.md

```diff
@@ -25,7 +25,11 @@ settings] you may want:
             $('#redactor').redactor({
                 plugins: ['uploadcare'],
                 uploadcare: {
-                    // see https://uploadcare.com/documentation/widget/#configuration
+                    // styling options
+                    buttonLabel: 'Image / file'
+                    buttonBefore: 'video',
```

Comment:
can you explain me what this does (it's guessable) and why "video" is default (this is not)?